app/src/main/java/com/zeapo/pwdstore/PasswordStore.kt

```diff
@@ -512,20 +533,16 @@ class PasswordStore : AppCompatActivity() {
                     }
                     AutofillMatcher.updateMatches(applicationContext, delete = filesToDelete)
                     item.file.deleteRecursively()
-                    adapter.remove(position)
-                    it.remove()
-                    adapter.updateSelectedItems(position, selectedItems)
```

Comment:
Can not updating on each deletion result in a situation where some files were deleted, others weren't, and the adapter never updates to remove the ones that are gone?